.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: c
+
+before_install:
+  - sudo apt-get -y install libcunit1 cmake
+addons:
+  apt:
+    update: true
+
+script:
+  - mkdir build && cd build
+  - cmake -DCMAKE_BUILD_TYPE=Debug ..
+  - make
```

Comment:
Also missing a step to execute the tests.

Reply:
wanted to know if it works at all (didn't!) but now its fine.